docs/develop/guides/cheat-sheets/momento-cache-python-cheat-sheet.md

```diff
@@ -0,0 +1,115 @@
+---
+sidebar_position: 1
+sidebar_label: Python
+sidebar_class_name: sidebar-item-python-cheat-sheet
+title: Cheat Sheet for Python + Momento Serverless Cache
+description: Everything to get you going with coding using Python and Momento Serverless Cache
+---
+
+# Cheat Sheet for Python with Momento Serverless Cache
+If you need to get going quickly with Python and Momento Serverless Cache, here are the basic API calls you'll need. When you need them, check out the [API reference page](../../api-reference/index.mdx) for more information on the full assortment of Momento API calls.
+
+If you combine all of the functions on this page into one python file, you'd have a central collection of functions you can import and call from other python code. In addition, if you are putting this into production, you might look to replace the print() calls with ones using the logging library (`import logging`) in python.
+
+## Import libraries and connect to return a CacheClient object
+```python
```

Comment:
Do you think you need a short description here to be consistent with the others?